src/objects/oo/zcl_abapgit_oo_class.clas.abap

```diff
@@ -195,6 +205,14 @@ CLASS zcl_abapgit_oo_class IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD get_method_includes.
+    " get method includes for implemented interfaces
+    " this will contain also leftover includes for deleted interface methods
+    rt_includes = cl_oo_classname_service=>get_all_method_includes( iv_classname ).
+    DELETE rt_includes WHERE cpdkey-cpdname NA '~'.
```

Comment:
I wonder if that still works if you use an alias in the method implementation (which you should not do but can if I recall correctly).

Reply:
i removed the delete since it should cover the alias case as well. worst case, it calls the repair unnecessarily.